db/people.yml

```diff
@@ -2450,4 +2450,62 @@ kknox:
       organization: Columbia University
       position: Graduate Researcher
   name: Kevin Knox
-  position: Vice President
\ No newline at end of file
+  position: Vice President
+ckkim:
+  active: false
+  aka:
```

Comment:
fyi you don't need to add a lot of aka's, we tend to add these organically as they are needed.  You can give a single entry, or just an empty list ``aka: []`` if you like.  It's also ok to add them...